internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -427,19 +427,19 @@ func (d *dataPlaneKafkaService) checkKafkaRequestCurrentStatus(kafka *dbapi.Kafk
 // stores routes reported by data plane to the database if not already persisted
 func (d *dataPlaneKafkaService) persistKafkaRoutes(kafka *dbapi.KafkaRequest, kafkaStatus *dbapi.DataPlaneKafkaStatus, cluster *api.Cluster) *serviceError.ServiceError {
 	if kafka.Routes != nil {
-		logger.Logger.V(10).Infof("skip persisting routes for Kafka %s as they are already stored", kafka.ID)
+		logger.Logger.V(10).Infof("skip persisting routes for Kafka %q as they are already stored", kafka.ID)
 		return nil
 	}
 
 	if len(kafkaStatus.Routes) < 1 {
-		logger.Logger.V(10).Infof("skip persisting routes for Kafka %s as they are not available", kafka.ID)
+		logger.Logger.V(10).Infof("skip persisting routes for Kafka %q as they are not available", kafka.ID)
 		return nil
 	}
 
-	logger.Logger.Infof("store routes information for kafka %s", kafka.ID)
+	logger.Logger.Infof("store routes information for kafka %q", kafka.ID)
 	clusterDNS, err := d.clusterService.GetClusterDNS(cluster.ClusterID)
 	if err != nil {
-		return serviceError.NewWithCause(err.Code, err, "failed to get DNS entry for cluster %s", cluster.ClusterID)
+		return serviceError.NewWithCause(err.Code, err, "failed to get DNS entry for ClusterID %q", cluster.ClusterID)
```

Comment:
```suggestion
		return serviceError.NewWithCause(err.Code, err, "failed to get DNS entry for cluster with ClusterID %q", cluster.ClusterID)
```

@miguelsorianod what do you think of this suggestion? To be applied through out if you agree.

Reply:
I am not going to apply this suggestion for now. The reason is that I think we need better terminology to differentiate between the kafka ID and the (OCM) cluster ID.

Even if you say `cluster with ClusterID` the following question arises: is the cluster the kafka Cluster that has a ClusterID assigned, or is it the (OCM) cluster ID itself?

Reply:
I am okay with any followup, a PR or a discussion on getting to a consistent use of the terminologies.

> Even if you say cluster with ClusterID the following question arises: is the cluster the kafka Cluster that has a ClusterID assigned, or is it the (OCM) cluster ID itself?

For this specific suggestion, I mean the later i.e  "is it the (OCM) cluster ID itself".

Reply:
I know. But I know because I've seen the logging code. But it is not clear just reading the text.